### PR TITLE
LCI 3 cards

### PR DESCRIPTION
**Pull request by Simisays**
Should wrap up LCI except the one remaining craft card

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/rumbling_rockslide.txt [hunk 1]
```diff
@@ -1,6 +0,0 @@
-Name:Rumbling Rockslide
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/rumbling_rockslide.txt:1]: need to restore this card

File: forge-gui/res/cardsfolder/upcoming/malamet_scythe.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Malamet Scythe
+ManaCost:2 G
+Types:Artifact Equipment
+K:Flash
+K:Equip:4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigAttach | TriggerDescription$ When CARDNAME enters the battlefield, attach it to target creature you control.
+SVar:TrigAttach:DB$ Attach | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature you control
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddPower$ 2 | AddToughness$ 2 | Add Description$ Equipped creature gets +2/+2.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/malamet_scythe.txt:8]: typo

File: forge-gui/res/cardsfolder/upcoming/rumbling_rockslide.txt (hunk 1 above)
Comment (paulsnoops) [reply forge-gui/res/cardsfolder/upcoming/rumbling_rockslide.txt:1]: Yeah I asked him to remove it because it's a duplicate

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/rumbling_rockslide.txt:1]: good catch :D

**Review by tool4ever — APPROVED** [review]

**Pull request merged**